src/test/steps/migrate_app_js.test.ts

```diff
@@ -308,6 +308,17 @@ describe("migrate app js", () => {
                      }`)
                   );
                 });
+                it("should correctly chain together invoke calls", async () => {
+                  writeFixtureSrc(`foo: function() {
+                    this.${api}("sharedWith").hide();
+                   }`);
+                  await subject(options);
+                  expect(readMigratedSrc()).to.have.string(
+                    wrapExpectedSrc(`foo: async function() {
+                       await wrapZafClient(this.zafClient, "${api}:sharedWith.hide"); 
```

Comment:
note for you to double check if this line works properly (see https://zendesk.slack.com/files/U1EHCH8PM/F7AMGF58S/disablesharingdemo.zip)